code/modules/mob/living/silicon/ai/say.dm

```diff
@@ -113,20 +112,20 @@
 	play_vox_word(message, character_to_use, emotion_to_use, src, src.z, null)
 
 
-/proc/play_vox_word(message, character, emotion, mob/living/speaker, z_level, mob/only_listener)
+/proc/play_vox_word(message, character, emotion, mob/living/silicon/ai/speaker, z_level, mob/only_listener)
 	var/api_url = "https://api.fifteen.ai/app/getAudioFile"
-
+	var/static/vox_voice_number = 0
 	var/datum/http_request/req = new()
-	var/vox_audio_number = rand(1,9999)
-	req.prepare(RUSTG_HTTP_METHOD_POST, api_url, "{\"character\":\"[character]\",\"text\":\"[message]\",\"emotion\":\"[emotion]\"}", list("Content-Type" = "application/json", "User-Agent" = "/tg/station 13 server"), json_encode(list("output_filename" = "data/vox_[vox_audio_number].wav")))
+	vox_voice_number++

```

Comment:
There is a reason why I suggested exactly what I wrote. If you do this, you risk that the play_vox_word proc will get called a second time before the first request is complete, resulting in a giant shitshow.

Reply:
oof, ya ill check it out

Reply:
This got resolved by accident